include/eve/module/real/core/function/regular/simd/x86/convert_128.hpp

```diff
@@ -78,6 +78,105 @@ namespace eve::detail
       }
     }
     //==============================================================================================
+    // Convert to 64 bits integer
+    //==============================================================================================
+    else if constexpr( std::is_integral_v<Out> && (sizeof(Out) == 8) )
+    {
+      //============================================================================================
+      // 8 -> 64 bits
+      //============================================================================================
+      if constexpr( std::is_integral_v<In> && (sizeof(In) == 1))
+      {
+        if constexpr( current_api >= sse4_1 )
+        {
+          if constexpr( std::is_signed_v<In> && (N::value <= 2) )
+          {
+            return _mm_cvtepi8_epi64(v0);
+          }
+          else if constexpr( !std::is_signed_v<In>  && (N::value <= 2) )
+          {
+            return _mm_cvtepu8_epi64(v0);
+          }
+          else
+          {
+            return convert_(EVE_RETARGET(simd_), v0, tgt);
+          }
+        }
+        else
+        {
+          return convert_(EVE_RETARGET(simd_), v0, tgt);
+        }
+      }
+      //============================================================================================
+      // 16 -> 64 bits
+      //============================================================================================
+      if constexpr( std::is_integral_v<In> && (sizeof(In) == 2))
+      {
+        if constexpr( current_api >= sse4_1 )
+        {
+          if constexpr( std::is_signed_v<In> && (N::value <= 2) )
+          {
+            return _mm_cvtepi16_epi64(v0);
+          }
+          else if constexpr( !std::is_signed_v<In>  && (N::value <= 2) )
+          {
+            return _mm_cvtepu16_epi64(v0);
+          }
+          else
+          {
+            return convert_(EVE_RETARGET(simd_), v0, tgt);
+          }
+        }
+        else
+        {
+          return convert_(EVE_RETARGET(simd_), v0, tgt);
+        }
+      }
+      //============================================================================================
+      // 32 -> 64 bits
+      //============================================================================================
+      if constexpr( std::is_integral_v<In> && (sizeof(In) == 4))
+      {
+        if constexpr( current_api >= sse4_1 )
```

Comment:
We have this code already, no? 
https://github.com/jfalcou/eve/blob/0e5d657d7ea13344e1ec3e31287efc3179e11b84/include/eve/module/real/core/function/regular/simd/x86/convert_128.hpp#L98

Let's reuse that